common/lib/xmodule/xmodule/course_module.py

```diff
@@ -834,6 +834,17 @@ class CourseFields(object):
         scope=Scope.settings,
     )
 
+    social_sharing_url = String(
```

Comment:
I'm wondering if this is something we want on the Course Descriptor, or if we want to continue adding anything to it in general. I'm not sure who currently owns the Course API / Course Details API designs at this point, but maybe it's worth seeing if there's a better place for Course metadata that doesn't querying the Course Descriptor.